tests/unit/Application/Assembler/ServiceStatusAssemblerTest.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Mockery as m;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+use Mockery\MockInterface;
+use Surfnet\ServiceProviderDashboard\Application\Assembler\ServiceStatusAssembler;
+use Surfnet\ServiceProviderDashboard\Application\Service\ServiceStatusService;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Service;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
+use Symfony\Component\Routing\RouterInterface;
+
+class ServiceStatusAssemblerTest extends MockeryTestCase
```

Comment:
An idea for an extra test: the mapState method of the assembler can throw an InvalidArgumentException. This behavior is currently untested.

Reply:
Removed the exception, this should never occur